gosu-test/src/test/gosu/gw/specContrib/statements/usesStatement/Errant_ExplicitStaticImportsAllowedByUnqualifiedClassNameFromTheSamePackage.gs

```diff
@@ -0,0 +1,15 @@
+package gw.specContrib.statements.usesStatement
```

Comment:
These files should be named Errant_ so that if in future, there happen to be an error in these, we can catch them in spec tests without the compilation failing.
There are many more files which are Errant and have only positive tests (i.e. no issue keys)